qcodes/station.py

```diff
@@ -36,6 +36,12 @@
 import qcodes.utils.validators as validators
 from qcodes.monitor.monitor import Monitor
 
+import tempfile
+from io import StringIO
+from collections import deque
+import ruamel.yaml
+from pathlib import Path
+from typing.Deque import deque as Tdeque
```

Comment:
This import looks strange. Are you sure it should not just be?

```
from typing import Deque as Tdeque
```

Reply:
I tried that first but my local mypy fails:
qcodes\station.py:702:10: error: Missing type parameters for generic type "Tdeque"

Reply:
But this is not much better since it fails with 

```
ModuleNotFoundError: No module named 'typing.Deque'; 'typing' is not a package
```

;)

What that error message means is that  Deque is a generic type like List so you should write
```
Tdeque[Any]
```
Since the definition of load is `(Union[Path, StreamTextType]) -> Any`

Reply:
For an unknown reason it passed my local mypy, although it is weird :).

I was looking for correct typing of the output, and Any should definitely work, so
"Tdeque[Any]" -> passes the mypy